src/main/webapp/resources/js/pages/analysis/components/jobErrors/GalaxyParameters.jsx

```diff
@@ -6,19 +6,19 @@
  */
 
 import React from "react";
-import { CodeBlock } from "../../../../components/typography";
+import { OutputWrapper } from "../../../../components/OutputFiles/OutputWrapper";
 
 export function GalaxyParameters({ galaxyJobErrors, currIndex }) {
   // Returns the galaxy parameters for the given index from the jobErrors object
   function getGalaxyParameters(index = 0) {
     return (
-      <CodeBlock>
+      <OutputWrapper overflowRequired={true} style={{ whiteSpace: "pre-wrap" }}>
```

Comment:
Do you think that the style should be internal to the component?  It sounds like something you would want each time.

Reply:
Updated in 02cc213ba4a9097b9f3be5383bd1a1a5f5973211